pkg/profiles/openapi_test.go

```diff
@@ -29,6 +31,9 @@ func TestSwaggerToServiceProfile(t *testing.T) {
 										},
 									},
 								},
+								VendorExtensible: spec.VendorExtensible{
+									Extensions: xLinkerdRetryable,
```

Comment:
I think it's more clear if you inline the extensions here.  e.g.

```
Extensions: {"x-linkerd-retryable": true},
```

Reply:
Done 👍